test/fielddef.test.ts

```diff
@@ -50,12 +49,15 @@ describe('fieldDef', () => {
     });
 
     it('should return correct title for aggregate', () => {
-      const fieldDef = {field: 'f', type: QUANTITATIVE, aggregate: AggregateOp.MEAN};
+      // if "aggregate: 'mean' as 'mean'" is changed to "aggregate: 'mean'", it won't pass the test because
+      // it will show that type of 'mean' is string instead of 'mean'
+      const fieldDef = {field: 'f', type: QUANTITATIVE, aggregate: 'mean' as 'mean'};
```

Comment:
@nung25649 Actually this is a valid point and seems annoying. 

However, removing `fieldDef` and just pass the object to the function works. 
```
assert.equal(title({field: 'f', type: QUANTITATIVE, aggregate: 'mean'}, {}), 'MEAN(f)');
```
(I guess before the function know that this object is a FieldDef type.)

Alternatively, we can add type FieldDef to 
```
const fieldDef: FieldDef = ...
```

but the former solution seems shorter.